src/vi/goctruyentranh/src/eu/kanade/tachiyomi/extension/vi/goctruyentranh/GocTruyenTranh.kt

```diff
@@ -143,10 +143,10 @@ class GocTruyenTranh : ParsedHttpSource(), ConfigurableSource {
                             .map { it.id }
                             .forEach { addQueryParameter("status", it) }
                     is ChapterCountList ->
-                        filter.state
-                            .filter { it.state }
-                            .map { it.id }
-                            .forEach { addQueryParameter("minChap", it) }
+                        {
+                            val chapterCount = getChapterCountList()[filter.state]
```

Comment:
consider adding function to `ChapterCountList` class to return id corresponding to current state

Reply:
> consider adding function to `ChapterCountList` class to return id corresponding to current state

I don't think that's necessary

Reply:
it would be a bit cleaner but sure w/e